docs/platform/howto/change-your-email-address.rst

```diff
@@ -1,23 +1,22 @@
 Change your email address
-=========================
+==========================
 
-It is not currently possible to change your login email address directly. It is possible, however, to migrate your Aiven resources to another email address.
+You can't change your login email address directly. Instead, you can create a new user with the new email address and then remove the user with the old email address. You must be an admin to do this.
 
 .. important::
     
-    Aiven support personnel cannot change your email address nor migrate projects between email addresses from support requests.
+    Aiven support cannot change your email address.
 
-Migrate Aiven projects between email addresses
-----------------------------------------------
+#. In the organization, click **Admin**.
 
-To start using an Aiven project from a new email address, invite your new email address to your existing project from the **Members** page of each project you wish to migrate:
+#. Click **Users**. 
 
-1. In the project, click **Members**.

```

Comment:
We should change that flow here, instead of inviting the user to the project the new user should be invited to the organization and added to the group.